packages/db/src/query/user.ts

```diff
@@ -182,17 +187,44 @@ export async function getUserDataByUserId(id: number) {
 }
 
 export async function deleteUser(id: number) {
-    return await db.delete(user).where(eq(user.id, id));
+    return db.delete(user).where(eq(user.id, id));
 }
 
 export async function getAllUsers() {
     return await db.select().from(user);
 }
 
 export async function setUserActive(id: number, isActive: boolean) {
-    return await db.update(user).set({ isActive }).where(eq(user.id, id));
+    return await db.update(user).set({isActive}).where(eq(user.id, id));
 }
 
 export async function setUserAdmin(id: number, isAdmin: boolean) {
-    return await db.update(user).set({ isAdmin }).where(eq(user.id, id));
+    return await db.update(user).set({isAdmin}).where(eq(user.id, id));
+}
+
+/**
+ * Get users with due report to create and send reports
+ * the report is due if the current date is greater than the last report date + interval or
+ * if the current date is equal to the last report date + interval and the current time is greater than the report time
```

Comment:
Wieder nur ne Kleinigkeit, aber die Beschreibung wäre nicht gut zu lesen, wenn man in der IDE über die Methode hovert. Da sollten mindestens Punkte drin sein und am besten auch Zeilenumbrüche (mit </br>)